package.json

```diff
@@ -618,16 +622,16 @@
         "viewsContainers": {
             "activitybar": [
                 {
-                    "id": "julia-explorer",
-                    "title": "Julia Explorer",
+                    "id": "julia-workspace",
+                    "title": "Julia Workspace",
                     "icon": "images/julia-dots.svg"
```

Comment:
I tried to unify workspace/REPL variable/explorer terminology, but let me know if you don't like this.